frontend/src/index.css

```diff
@@ -80,11 +80,48 @@ select {
 }
 
 body {
-    background-color: #f7f8fc;
+    background-color: #e7e7e7;
     font-family: Muli;
     font-style: normal;
     font-weight: normal;
     font-size: 20px;
     line-height: 26px;
     color: #424147;
 }
+
+/* Style the navigation menu */
+.navbar {
+    width: 100%;
+    background-color: #466e78;
+    overflow: auto;
+  }
+  
+  /* Navigation links */
+  .navbar a {
+    float: left;
+    padding: 12px;
+    color: white;
+    text-decoration: none;
+    font-size: 17px;
+    width: 20%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
+    text-align: right; /* If you want the text to be centered */
+  }
+  .navbar b {
+    float: left;
+    padding: 12px;
+    color: white;
+    text-decoration: none;
+    font-size: 17px;
+    width: 60%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
+    text-align: left; /* If you want the text to be centered */
+  }
+   
+  /* Add responsiveness - on screens less than 500px, make the navigation links appear on top of each other, instead of next to each other */
+  @media screen and (max-width: 500px) {
+    .navbar a {
+      float: none;
+      display: block;
+      width: 100%;
+      text-align: left; /* If you want the text to be left-aligned on small screens */
+    }
+  }
```

Comment:
for improved readability, we also might want to break up the index.css file into some grouped css files. what do you think about that? @gandompm

Reply:
ja, would be better than this.